gcp/table_gcp_monitoring_alert_policy.go

```diff
@@ -0,0 +1,186 @@
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "notification_channels",
+				Description: "Identifies the notification channels to which notifications should be sent when incidents are opened or closed or when new violations occur on an already opened incident.",
+				Type:        proto.ColumnType_JSON,
+			},
+			{
+				Name:        "user_labels",
+				Description: "User-supplied key/value data to be used for organizing and identifying the AlertPolicy objects.",
+				Type:        proto.ColumnType_JSON,
+			},
+
+			// standard steampipe columns
+			{
+				Name:        "title",
+				Description: ColumnDescriptionTitle,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("DisplayName"),
+			},
+			{
+				Name:        "tags",
+				Description: ColumnDescriptionTags,
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("UserLabels"),
+			},
+			{
+				Name:        "akas",
+				Description: ColumnDescriptionAkas,
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromP(getMonitoringAlertPolicyAka, "Akas"),
+			},
+
+			// standard gcp columns
+			{
+				Name:        "location",
+				Description: ColumnDescriptionLocation,
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromConstant("global"),
+			},
+			{
+				Name:        "project",
+				Description: ColumnDescriptionProject,
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getProject,
+				Transform:   transform.FromValue(),
+			},
+		},
+	}
+}
+
+//// LIST FUNCTION
+
+func listMonitoringAlertPolicy(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
+	// Create Service Connection
+	service, err := MonitoringService(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+
+	// Get project details
+	projectData, err := activeProject(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+	project := projectData.Project
+
+	resp := service.Projects.AlertPolicies.List("projects/" + project)
+
+	if err := resp.Pages(ctx, func(page *monitoring.ListAlertPoliciesResponse) error {
+		for _, alertPolicy := range page.AlertPolicies {
+			d.StreamListItem(ctx, alertPolicy)
+		}
+		return nil
+	}); err != nil {
+		return nil, err
+	}
+
+	return nil, err
+}
+
+//// HYDRATE FUNCTIONS
+
+func getMonitoringAlertPolicy(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	plugin.Logger(ctx).Trace("getMonitoringAlertPolicy")
+
+	// Get project details
+	projectData, err := activeProject(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+	project := projectData.Project
+	name := d.KeyColumnQuals["name"].GetStringValue()
+
+	// Create Service Connection
+	service, err := MonitoringService(ctx, d)
+	if err != nil {
+		return nil, err
+	}
+
+	req, err := service.Projects.AlertPolicies.Get("projects/" + project + "/alertPolicies/" + name).Do()
```

Comment:
Is it correct?
As per the api `Name` includes both projectID and resourceID
![image](https://user-images.githubusercontent.com/38218418/114542126-29b88f80-9c75-11eb-9dce-808c0634a19c.png)